src/workflows-app/SubmissionConfig.test.js

```diff
@@ -135,6 +135,17 @@ describe('BaseSubmissionConfig renders workflow details', () => {
     expect(screen.getAllByText('Select a data table')[0]).toBeInTheDocument();
     expect(screen.getByText('FOO')).toBeInTheDocument();
 
+    // Call cache button should exist, be checked by default, and toggle to false when clicked.
+    expect(screen.getByText('Call Caching:')).toBeInTheDocument();
+    const callCacheToggleButton = screen.getByTestId('call-cache-toggle');
+    expect(callCacheToggleButton).toBeDefined();
+    expect(callCacheToggleButton).toHaveProperty('checked', true);
+    act(() => {
+      callCacheToggleButton.click();
+    });
```

Comment:
```suggestion
    await user.click(callCacheToggleButton);
```

May need to add a line at the top of the test
```js
const user = userEvent.setup();
```